Client/Frontend/Browser/Toolbars/BottomToolbar/Menu/Bookmarks/BookmarkFormFieldsProtocol.swift

```diff
@@ -0,0 +1,40 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Shared
+
+protocol BookmarkFormFieldsProtocol where Self: UIView {
```

Comment:
I think this is a bit more clear ([and safer](https://developer.apple.com/documentation/xcode_release_notes/xcode_10_2_release_notes/swift_5_release_notes_for_xcode_10_2) apparently): 
```suggestion
protocol BookmarkFormFieldsProtocol: UIView {
```